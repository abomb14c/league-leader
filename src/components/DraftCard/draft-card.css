.draft-card {
  align-items: center;
  border-top: 1px solid #C63D0F;
  background-color: #3B3738;
  display: flex;
  height: 42px;
  text-align: left;
  padding-left: 2%;
}

.draft-name {
  font-weight: 200;
  color: #FDF3E7;
}

.draft-pick {
  border-bottom: 1px solid #C63D0F;
}