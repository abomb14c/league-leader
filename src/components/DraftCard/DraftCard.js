import React from 'react';
import './draft-card.css';

export const DraftCard = ({name}) => {
  return (
    <div className="draft-card">
      <h3 className="draft-name">{name}</h3>
    </div>
  )
}


export default DraftCard;