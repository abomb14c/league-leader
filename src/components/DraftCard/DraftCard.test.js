import React from 'react';
import { shallow } from 'enzyme';
import  DraftCard  from './DraftCard';

describe('DraftCard', () => {
  let wrapper;
  let mockProps;

  beforeEach(() => {
    mockProps = {
      name: 'alan'
    };

    wrapper = shallow(<DraftCard 
      {...mockProps}
    />);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});