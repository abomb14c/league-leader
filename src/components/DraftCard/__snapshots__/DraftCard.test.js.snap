// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DraftCard should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    name="alan"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3
        className="draft-name"
>
        alan
</h3>,
      "className": "draft-card",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "alan",
        "className": "draft-name",
      },
      "ref": null,
      "rendered": "alan",
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3
          className="draft-name"
>
          alan
</h3>,
        "className": "draft-card",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "alan",
          "className": "draft-name",
        },
        "ref": null,
        "rendered": "alan",
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
